src/participant/prompts/query.ts

```diff
@@ -5,56 +5,62 @@ import { getHistoryMessages } from './history';
 import { getStringifiedSampleDocuments } from '../sampleDocuments';
 
 export class QueryPrompt {
-  static async getAssistantPrompt({
-    databaseName = 'mongodbVSCodeCopilotDB',
-    collectionName = 'test',
-    schema,
-    sampleDocuments,
-  }: {
-    databaseName?: string;
-    collectionName?: string;
-    schema?: string;
-    sampleDocuments?: Document[];
-  }): Promise<vscode.LanguageModelChatMessage> {
-    let prompt = `You are a MongoDB expert.
-
-Your task is to help the user craft MongoDB queries and aggregation pipelines that perform their task.
+  static getAssistantPrompt(): vscode.LanguageModelChatMessage {
+    const prompt = `You are a MongoDB expert.
+Your task is to help the user craft MongoDB shell syntax code to perform their task.
 Keep your response concise.
-You should suggest queries that are performant and correct.
-Respond with markdown, suggest code in a Markdown code block that begins with \`\`\`javascript and ends with \`\`\`.
-You can imagine the schema.
+You must suggest code that is performant and correct.
+Respond with markdown, write code in a Markdown code block that begins with \`\`\`javascript and ends with \`\`\`.
 Respond in MongoDB shell syntax using the \`\`\`javascript code block syntax.
```

Comment:
[nit] This is a bit repetitive. Perhaps it's useful to reinforce the message, but I imagine just one sentence explaining the markdown requirements should be enough.

Reply:
I think you're probably right here, we also had two before, so if it's alright with you I'm going to not to change it in this pr. It seems like a less glaring issue, once we have some more accuracy tests I'd like to investigate this more.